src/test/java/outcomes/process_executor/go/no_find_executable_file_by_directory/NoFindExecutableFileByDirectory.java

```diff
@@ -0,0 +1,27 @@
+package outcomes.process_executor.go.no_find_executable_file_by_directory;
```

Comment:
I wrote tests only for `GoExecutor`, since the behavior for the `PythonExecutor`, `JaaScriptExecutor` and `ShellExecutor` is same